app/components/status_page_component.rb

```diff
@@ -0,0 +1,35 @@
+class StatusPageComponent < BaseComponent
+  ICONS = {
+    warning: [],
+    error: [:lock],
+  }.freeze
+
+  renders_one :header, ::PageHeadingComponent
+  renders_many :action_buttons, ->(**button_options) do
+    ButtonComponent.new(**button_options, big: true, wide: true)
+  end
+  renders_one :troubleshooting_options
```

Comment:
Future work should adapt [`app/views/shared/_troubleshooting_options.html.erb`](https://github.com/18F/identity-idp/blob/main/app/views/shared/_troubleshooting_options.html.erb) into a component, so this could be implemented as a pass-through similar to `PageHeadingComponent` and `ButtonComponent` above.

```suggestion
  renders_one :troubleshooting_options, TroubleshootingOptionsComponent
```